README.md

```diff
@@ -376,19 +374,20 @@ you chain more over the same structure. It's therefore recommended that you use
 traverse a large list (10000+ items).
 
 #### ap
-##### `#ap :: Future a b ~> Future a (b -> c) -> Future a c`
+##### `#ap :: Future a (b -> c) ~> Future a b -> Future a c`
```

Comment:
I'm not sure if `fantasy-land/ap` deserves a type signature. I believe that conceptually it's not there, and it's functions like `Z.ap` which are polymorphic.